### PR TITLE
eCloud VPN commands

### PR DESCRIPTION
**Pull request by 0x4c6565**
This PR adds VPN endpoint/session/service/profile group commands

### CODE REVIEW

File: cmd/ecloud/ecloud_vpnendpoint.go [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+	return output.CommandOutput(cmd, OutputECloudVPNEndpointsProvider(endpoints))
+}
+
+func ecloudVPNEndpointShowCmd(f factory.ClientFactory) *cobra.Command {
+	return &cobra.Command{
+		Use:     "show <endpoint: id>...",
+		Short:   "Shows a VPN endpoint",
+		Long:    "This command shows one or more VPN endpoints",
+		Example: "ukfast ecloud vpnendpoint show vpns-abcdef12",
```
Comment (Overglazed) [inline cmd/ecloud/ecloud_vpnendpoint.go:65]: Examples should use prefix `vpne-` for vpn endpoints

File: cmd/ecloud/ecloud_vpnendpoint_test.go [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+}
+
+func Test_ecloudVPNEndpointShow(t *testing.T) {
+	t.Run("SinglePolicy", func(t *testing.T) {
+		mockCtrl := gomock.NewController(t)
+		defer mockCtrl.Finish()
+
+		endpoint := mocks.NewMockECloudService(mockCtrl)
+
+		endpoint.EXPECT().GetVPNEndpoint("vpns-abcdef12").Return(ecloud.VPNEndpoint{}, nil).Times(1)
```
Comment (Overglazed) [inline cmd/ecloud/ecloud_vpnendpoint_test.go:78]: tests should use `vpne-` for vpn endpoints

**Review by Overglazed — APPROVED** [review]
LGTM! :)

A couple resource prefixes just need updating to match the resource

File: cmd/ecloud/ecloud_vpnendpoint.go (hunk 1 above)
Comment (0x4c6565) [reply cmd/ecloud/ecloud_vpnendpoint.go]: Good spot, sorted :)

File: cmd/ecloud/ecloud_vpnendpoint_test.go (hunk 2 above)
Comment (0x4c6565) [reply cmd/ecloud/ecloud_vpnendpoint_test.go]: Good spot, sorted :)

**Pull request merged**